gtsam/discrete/DecisionTree-inl.h

```diff
@@ -307,7 +307,7 @@ namespace gtsam {
 #ifndef NDEBUG
       typename Assignment<L>::const_iterator it = x.find(label_);
       if (it == x.end()) {
-        std::cout << "Trying to find value for " << label_ << std::endl;
+        // std::cout << "Trying to find value for " << label_ << std::endl;
```

Comment:
not expecting gtsam changes other than hybrid in this PR...

Reply:
Gotcha. Undoing this.